xcat-inventory/xcclient/inventory/shell.py

```diff
@@ -39,10 +39,11 @@ def add_subcommands(self, subparsers, revision):
     @shell.arg('-s','--schema-version',dest='version', metavar='<version>', help='schema version of the inventory file. Valid schema versions: '+','.join(mgr.InventoryFactory.getAvailableSchemaVersions())+'. '+'If not specified, the "latest" schema version will be used')
     @shell.arg('--dry', dest='dryrun', action="store_true", default=False, help='Dry run mode, nothing will be commited to xcat database')
     @shell.arg('-c','--clean', dest='update', action="store_false", default=True, help='clean mode. IF specified, all objects other than the ones to import will be removed')
+    @shell.arg('-e','--env', dest='env', metavar='<env_var>', action='append', help='environment variables in osimage')
```

Comment:
should better be `help='the values of variables in object definitions(only available for osimage object), syntax: "<variable name>=<variable value>" , this option can be used multiple times to specify multiple variables'`